src/main/java/project/myblog/web/dto/post/PostResponse.java

```diff
@@ -0,0 +1,65 @@
+package project.myblog.web.dto.post;
+
+import project.myblog.domain.Post;
+
+import java.util.Objects;
+
+public class PostResponse {
+    private Long id;
+    private String title;
+    private String contents;
+    private String author;
+
+    public PostResponse(Post post) {
+        this.id = post.getId();
+        this.title = post.getTitle();
+        this.contents = post.getContents();
+        this.author = post.getMember().getEmail();
+    }
```

Comment:
```suggestion
    public PostResponse(Post post) {
        this(post.getId(), post.getTitle(), post.getContents(), post.getEmail());
    }
```
생성자 하나를 주 생성자로 만들어 사용하는 것이 더 좋을 것 같습니다.